src/Features/Core/Portable/MoveToNamespace/MoveToNamespaceAnalysisResult.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.MoveToNamespace
+{
+    internal partial class MoveToNamespaceAnalysisResult
+    {
+        public bool CanPerform { get; }
+        public Document Document { get; }
+        public string ErrorMessage { get; }
+        public SyntaxNode Container { get; }
+        public string OriginalNamespace { get; }
+        public ContainerType Type { get; }
+
+        public MoveToNamespaceAnalysisResult(
+            Document document,
+            SyntaxNode container,
+            string originalNamespace,
+            ContainerType containerType)
+        {
+            CanPerform = true;
+            Document = document;
+            Container = container;
+            OriginalNamespace = originalNamespace;
+            Type = containerType;
+        }
+
+        public MoveToNamespaceAnalysisResult(string errorMessage)
```

Comment:
The error message should only matter if you're actually going to display it to the user, which I thought would only happen in the command handler case where we show the dialog. If it's just a lightbulb item, then there's just no lightbulb item (with no explanation).

Reply:
This is a pattern I've seen before, and assumed it was just for diagnostic purposes. Probably not needed